prisma/schema.prisma

```diff
@@ -8,150 +8,101 @@ datasource db {
 }
 
 model User {
-  uuid String @unique
-  password String
-
-  username String @unique
-  email String @unique
-
-  firstName String
-  lastName String
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
+  id        Int      @id @default(autoincrement()) //dont want to give this info to the user
+  uuid      String   @unique @default(uuid())
+  password  String
+  username  String   @unique
+  email     String   @unique
+  firstName String   @map("first_name") //dbs will have snake case
+  lastName  String   @map("last_name")
+  avatar    String
+  type      Role     @default(USER)
+  createdAt DateTime @default(now()) @map("created_at")
+  updatedAt DateTime @updatedAt @map("updated_at")
+  orders    Order[] //1:M
+  vendor    Vendor? //1:1
 
-  type Role @default(USER)
-  id Int @id @default(autoincrement())
+  @@map("users")
 }
 
-model Movie {
-  uuid        String   @unique
-  name        String
-  createdAt   DateTime @default(now())
-  updatedAt   DateTime @updatedAt
-  id          Int      @id @default(autoincrement())
-  images      String[]
-  description String
-  genreId     Int      @map("genre_id")
-  genre       Genre    @relation(fields: [genreId], references: [id])
+model Vendor {
+  id             Int             @id @default(autoincrement())
+  uuid           String          @unique @default(uuid())
+  userId         Int             @unique @map("user_id")
+  name           String          @unique @map("vendor_name")
+  createdAt      DateTime        @default(now()) @map("created_at")
+  updatedAt      DateTime        @updatedAt @map("updated_at")
+  user           User            @relation(fields: [userId], references: [id]) //userId here should match the id in the User model
+  vendorProducts VendorProduct[]
```

Comment:
I don't see the table name decorator for some cases, please let's add it @@map("vendors")